pt_BR.ini

```diff
@@ -597,28 +600,31 @@ written = Escrito em C++ pela velocidade e portabilidade
 [Reporting]
 Bad = Ruim
 FeedbackDelayInfo = Seus dados estão sendo enviados em segundo plano.
-FeedbackDesc = Como está a emulação? Conte para nós e para a comunidade!
+FeedbackDesc = Como está a emulação? Conte para nós!
+FeedbackIncludeScreen = Incluir uma captura de tela
 FeedbackSubmitDone = Seus dados foram enviados.
-FeedbackSubmitFail = Não foi possível enviar dados para o servidor. Tente atualizar o PPSSPP.
+FeedbackSubmitFail = Erro ao enviar dados. Atualize o PPSSPP.
 FeedbackThanks = Obrigado pelo seu feedback!
 Gameplay = Jogabilidade
 Graphics = Gráficos
 Great = Ótimo
-In-game = Jogo inicia apresenta bugs demais pra poder jogar
+In-game = Inicia/Bugs
+In-game Description = Jogo inicia porém é impossível jogar
 Menu/Intro = Menu/Intro
-Menu/Intro Description = "Não pode nem entrar no jogo""
-Nothing = Nada
-Nothing Description = Completamente quebrado
+Menu/Intro Description = Trava no menu/intro
+Nothing = Injogável
```

Comment:
Yeah, this seems better.  Maybe "Nothing" is bad in English, but people will often say "nothing works" or "it goes nowhere" to mean something is completely broken.  Sorry for the misleading translation key.

-[Unknown]